### ISSUE TITLE
Remove the `show_prev_next` footer?

### ISSUE THREAD

**Issue by JoeZiminski** [neuroinformatics-unit/HowTo#54]
Personally I'm not a fan of the show prev / next footer as (imo) it duplicates the information from the left-sidebar and can make things a little cluttered. What do people think? If there is a consensus we can kill it with

```
html_theme_options = {
  "show_prev_next": False
}
```
in the `conf.py`!

**Comment by adamltyson** [comment]
I'm ambivalent, but I think it's the least use on this site, as there are very few (no?) sequential pages where this makes sense.

**Comment by niksirbi** [comment]
I agree with Adam. It's often useful when the pages have a sequential logic to them, but that's not the case in this website, so fine with removing it.

**Comment by adamltyson** [comment]
Fixed with https://github.com/neuroinformatics-unit/HowTo/pull/56
